docs/src/transformations.md

```diff
@@ -13,42 +13,301 @@ transform
 tn = TensorNetwork(circuit)
-transformed_tn = transform(tn, Tenet.RankSimplification)
+
+# Apply transformations to the tensor network
+transformed_tn = transform(tn, [Tenet.AntiDiagonalGauging, Tenet.DiagonalReduction, Tenet.ColumnReduction, Tenet.RankSimplification])
 
 fig = Figure() # hide
-ax1 = Axis(fig[1, 1]; title="Original TensorNetwork") # hide
-p1 = plot!(ax1, tn; node_size=5.) # hide
-ax2 = Axis(fig[1, 2], title="Transformed TensorNetwork") # hide
-p2 = plot!(ax2, transformed_tn; node_size=5.) # hide
-ax1.titlesize=20 # hide
-ax2.titlesize=20 # hide
+ax1 = Axis(fig[1, 1]) # hide
+p1 = plot!(ax1, tn; edge_width=0.75, node_size=8., node_attr=(strokecolor=:black, strokewidth=0.5)) # hide
+ax2 = Axis(fig[1, 2]) # hide
+p2 = plot!(ax2, transformed_tn; edge_width=0.75, node_size=8., node_attr=(strokecolor=:black, strokewidth=0.5)) # hide
+ax1.titlesize, ax2.titlesize = 20, 20 # hide
+hidedecorations!(ax1) # hide
+hidespines!(ax1) # hide
+hidedecorations!(ax2) # hide
+hidespines!(ax2) # hide
+
+Label(fig[1, 1, Bottom()], "Original") # hide
+Label(fig[1, 2, Bottom()], "Transformed") # hide
+
 fig # hide
 ```
 
 # Transformations
 
+## Hyperindex converter
 ```@docs
 Tenet.HyperindConverter
+```
+
+## Diagonal reduction
+```@example plot
+set_theme!(resolution=(800,200)) # hide
+fig = Figure() #hide
+
+data = zeros(Float64, 2, 2, 2, 2) #hide
+for i in 1:2 #hide
+    for j in 1:2 #hide
+        for k in 1:2 #hide
+            data[i, i, j, k] = k #hide
+        end #hide
+    end #hide
+end #hide
+
+A = Tensor(data, (:i, :j, :k, :l)) #hide
+B = Tensor(rand(2, 2), (:i, :m)) #hide
+C = Tensor(rand(2, 2), (:j, :n)) #hide
+
+tn = TensorNetwork([A, B, C]) #hide
+reduced = transform(tn, Tenet.DiagonalReduction) #hide
+
+smooth_annotation!( #hide
+    fig[1, 1]; #hide
+    color = bg_blue, #hide
+    xlims = [-2, 2], #hide
+    ylims = [-2, 2], #hide
+    offset_x = -0.21, #hide
+    offset_y = -0.42, #hide
+    radius_x = 0.38, #hide
+    radius_y = 0.8, #hide
+    num_waves = 6, #hide
+    fluctuation_amplitude = 0.02, #hide
+    phase_shift = 0.0) #hide
+plot!(fig[1, 1], tn, layout=Spring(iterations=1000, C=0.5, seed=100); node_color=[red, orange, orange, :black, :black,:black, :black]) #hide
+
+smooth_annotation!( #hide
+    fig[1, 2]; #hide
+    color = bg_blue, #hide
+    xlims = [-2, 2], #hide
+    ylims = [-2, 2], #hide
+    offset_x = 0.1, #hide
+    offset_y = -0.35, #hide
+    radius_x = 0.38, #hide
+    radius_y = 1.1, #hide
+    num_waves = 5, #hide
+    fluctuation_amplitude = 0.02, #hide
+    phase_shift = 1.9) #hide
+plot!(fig[1, 2], reduced, layout=Spring(iterations=1000, C=0.5, seed=100),  node_color=[orange, orange, red, :black, :black, :black, :black, :black]) #hide
+
+Label(fig[1, 1, Bottom()], "Original") #hide
+Label(fig[1, 2, Bottom()], "Transformed") #hide
+
+fig #hide
+```
+
+```@docs
 Tenet.DiagonalReduction
-Tenet.RankSimplification
+```
+
```

Comment:
```suggestion
```@example plot
set_theme!(resolution=(800,200)) # hide
fig = Figure() #hide

data = zeros(Float64, 2, 2, 2, 2) #hide
for i in 1:2 #hide
    for j in 1:2 #hide
        for k in 1:2 #hide
            data[i, i, j, k] = k #hide
        end #hide
    end #hide
end #hide

A = Tensor(data, (:i, :j, :k, :l)) #hide
B = Tensor(rand(2, 2), (:i, :m)) #hide
C = Tensor(rand(2, 2), (:j, :n)) #hide

tn = TensorNetwork([A, B, C]) #hide
reduced = transform(tn, Tenet.DiagonalReduction) #hide

smooth_annotation!( #hide
    fig[1, 1]; #hide
    color = bg_blue, #hide
    xlims = [-2, 2], #hide
    ylims = [-2, 2], #hide
    offset_x = -0.21, #hide
    offset_y = -0.42, #hide
    radius_x = 0.38, #hide
    radius_y = 0.8, #hide
    num_waves = 6, #hide
    fluctuation_amplitude = 0.02, #hide
    phase_shift = 0.0) #hide
plot!(fig[1, 1], tn, layout=Spring(iterations=1000, C=0.5, seed=100); node_color=[red, orange, orange, :black, :black,:black, :black]) #hide

smooth_annotation!( #hide
    fig[1, 2]; #hide
    color = bg_blue, #hide
    xlims = [-2, 2], #hide
    ylims = [-2, 2], #hide
    offset_x = 0.1, #hide
    offset_y = -0.35, #hide
    radius_x = 0.38, #hide
    radius_y = 1.1, #hide
    num_waves = 5, #hide
    fluctuation_amplitude = 0.02, #hide
    phase_shift = 1.9) #hide
plot!(fig[1, 2], reduced, layout=Spring(iterations=1000, C=0.5, seed=100),  node_color=[orange, orange, red, :black, :black, :black, :black, :black]) #hide

Label(fig[1, 1, Bottom()], "Original") #hide
Label(fig[1, 2, Bottom()], "Transformed") #hide

fig #hide
```